app/api/ada/index.js

```diff
@@ -232,7 +232,7 @@ export type GetTransactionsRequestOptions = {
   limit: number,
 };
 export type GetTransactionsRequest = {
-  ...$Shape<GetTransactionsRequestOptions>,
+  ...Inexact<GetTransactionsRequestOptions>,
```

Comment:
is it possible to make `GetTransactionsRequestOptions` exact rather than make this Inexact?